spectrocloud/resource_registry_oci_ecr.go

```diff
@@ -30,65 +30,78 @@ func resourceRegistryOciEcr() *schema.Resource {
 		SchemaVersion: 2,
 		Schema: map[string]*schema.Schema{
 			"name": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The name of the OCI registry.",
 			},
 			"type": {
 				Type:         schema.TypeString,
 				Required:     true,
 				ForceNew:     true,
 				ValidateFunc: validation.StringInSlice([]string{"ecr", "basic"}, false),
+				Description:  "The type of the registry. Possible values are 'ecr' (Amazon Elastic Container Registry) or 'basic' (for other types of registries).",
 			},
 			"is_private": {
-				Type:     schema.TypeBool,
-				Required: true,
+				Type:        schema.TypeBool,
+				Required:    true,
+				Description: "Specifies whether the registry is private or public. Private registries require authentication to access.",
 			},
 			"endpoint": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The URL endpoint of the OCI registry. This is where the container images are hosted and accessed.",
 			},
 			"provider_type": {
-				Type:     schema.TypeString,
-				Optional: true,
-				Default:  "helm",
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "helm",
+				Description: "The type of provider used for interacting with the registry. The default is 'helm'.",
 			},
 			"credentials": {
-				Type:     schema.TypeList,
-				Required: true,
-				MaxItems: 1,
+				Type:        schema.TypeList,
+				Required:    true,
+				MaxItems:    1,
+				Description: "Authentication credentials for accessing the OCI registry.",
```

Comment:
```suggestion
				Description: "Authentication credentials to access the private OCI registry. Required if `is_private` is set to `true`",
```
Minor language touchup.